x-pack/plugins/ingest_manager/server/services/agents/unenroll.ts

```diff
@@ -15,9 +17,7 @@ export async function unenrollAgents(
   const response = [];
   for (const id of toUnenrollIds) {
     try {
-      await soClient.update<AgentSOAttributes>(AGENT_SAVED_OBJECT_TYPE, id, {
-        active: false,
-      });
+      await unenrollAgent(soClient, id);
```

Comment:
What happens if the unenrollment was not successful? Will the user be able to try again?

Reply:
Yes the user will be able to try again and you can call multiple time ES to invalidate api key even if the API key is already invalidated.